src/2XXXR0_a_motivation_and_plan_for_a_physical_quantities_and_units_library.md

```diff
@@ -150,6 +150,8 @@ different libraries get means to communicate with each other.
 Mission/life-critical projects or those for embedded devices often can only use external libraries
 that are officially certified (i.e. MISRA). This is a huge issue for those companies as those
 typically are natural users of physical quantities and units library.
+<!-- I'm having trouble understanding the previous paragraph.
+IIUC, those certified libraries have interfaces with units, but are not type-safe. -->
 
```

Comment:
My bad. I should explain those better. According to my knowledge, none of the physical units libraries are currently MISRA-certified (maybe only Boost.Units as a part of Boost but I doubt it as well). For sure they can't use mp-units.